.eslintrc.json

```diff
@@ -14,7 +14,7 @@
     },
     "rules": {
         "linebreak-style": ["off"],
-        "complexity": ["error", {"max": 3}],
+        "complexity": ["error", {"max": 4}],
```

Comment:
Why do you need a complexity limit of 4? Looks like the code doesn't have such structures?

Reply:
I changed this in one of the previous commits as it was not working with 3, but at this point in time it is working so I'll revert it back to 3. Thank you for your suggestion. @numocityadmin @sudeeprp

Reply:
@numocityadmin @sudeeprp I can know what i have to next or should i post a proposal after this. Thank you.

Reply:
@numocityadmin  @sudeeprp Sir, only a few days are left can you please tell me what to do.